solr/solrj/src/java/org/apache/solr/common/util/SimpleOrderedMap.java

```diff
@@ -86,6 +91,79 @@ public static SimpleOrderedMap<Object> of() {
    * @return SimpleOrderedMap containing one key-value pair
    */
   public static <T> SimpleOrderedMap<T> of(String name, T val) {
-    return new SimpleOrderedMap<>(List.of(name, val));
+    return new SimpleOrderedMap<T>(List.of(name, val));
+  }
+
```

Comment:
minor: (organizational) better to place these instance methods above the static methods that you recently added